scripter.py

```diff
@@ -43,6 +43,17 @@ def __linux_create_wallpaper_script(pokemon):
     if "gnome" in os.environ.get("GDMSESSION"):
         fmt = 'gsettings set org.gnome.desktop.background picture-uri "file://{}"'
         return fmt.format(pokemon.get_path())
+    
+    if "xubuntu" in os.environ.get("GDMSESSION"):
```

Comment:
What if the user is running Xfce on another distro than Xubuntu?

You might want to use an elif here too.